README.md

```diff
@@ -145,36 +146,37 @@ linkStyle default opacity:0.5
   preferences_controller --> controller_utils;
   preferences_controller --> keyring_controller;
   profile_sync_controller --> base_controller;
+  profile_sync_controller --> keyring_controller;
+  profile_sync_controller --> accounts_controller;
+  profile_sync_controller --> network_controller;
   queued_request_controller --> base_controller;
   queued_request_controller --> controller_utils;
-  queued_request_controller --> json_rpc_engine;
```

Comment:
Hmm, when I run `yarn update-readme-content`, it restores this.

Maybe the graph changed again since you made this commit

Reply:
Non-blocking, as I see that this graph is already stale on `main`, but this would be nice to fix

Reply:
may have been bad PR splitting on my part

Reply:
Addressed in latest: https://github.com/MetaMask/core/pull/4812/commits/738c29177e159f8465a587bd2de50ba069320c03